pkg/transform/transform_test.go

```diff
@@ -15,6 +16,9 @@ import (
 )
 
 func TestOauthGenYAML(t *testing.T) {
+	env.Config().Set("Manifests", true)
+	env.Config().Set("Reporting", true)
```

Comment:
Not sure if we need reports in such test.

Reply:
Good catch, thanks

Reply:
That said it fails without setting the reports. I'll leave for now.